app.js

```diff
@@ -162,8 +166,15 @@ async function refreshMetrics() {
   } catch (e) {
     console.log("there was an error on request3 ", e)
   }
+try {
+  const rawResponse4 = await nhClient.getExchangeRates()
+  const data4 = rawResponse4.data
+  //console.log(data4)
+  btcEurRate.set(+data4['BTCEURS'])
```

Comment:
Can't we just move line 173 under line 165? And then we don't need the whole request block.

Reply:
I'm not javascript expert, so it's definitely better/easier way to implement. Would be also great to have rates as variable filled from docker enviroment to be flexible and not hardcoded.